lib/vanagon/platform/dsl.rb

```diff
@@ -51,6 +51,7 @@ def platform(name, &block)
                     end
 
         yield(self)
+        environment 'VANAGON_PLATFORM', platform_name.tr('-', '.')
```

Comment:
This is intended to be used in metric rollup; nothing else has used this value inside the Makefile, so I'm not *super* worried about it. Open to leaving it alone, or even renaming this variable and using `VANAGON_PLATFORM` for the origina value. @underscorgan, @shrug, @branan feedback? Thoughts?